### PR TITLE
feat(typescript): prevent duplicate exports

### PR DESCRIPTION
**Pull request by masoudmanson**
## Summary

**Structural Element (Base, Gene, DNA, Chromosome or Cell)**
Shortcut ticket: [sh-223901](https://app.shortcut.com/sci-design-system/story/223901/preventing-duplicate-type-names-in-the-library)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7768,9 +7768,13 @@ cz-conventional-changelog@^3.3.0:
   optionalDependencies:
     "@commitlint/load" ">6.1.1"
 
-"czifui@file:czifui-v0.0.21-0.tgz":
-  version "0.0.21-0"
-  resolved "file:czifui-v0.0.21-0.tgz#9edb11babc29b8c0d8dc8e55db107f3f677236b6"
+"czifui@file:czifui-v0.0.23.tgz":
```
Comment (tihuan) [inline yarn.lock:7771]: Just curious, if package.json version is 0.0.24, where did yarn.lock get 0.0.33 from? Do we need this or just 0.0.24 😄 ?

Thanks so much, Masoud!

**Review by tihuan — APPROVED** [review]
LGTM! Thanks for the solution, @masoudmanson 🙏 Just one quick question 🚀 !

File: yarn.lock (hunk 1 above)
Comment (masoudmanson) [reply yarn.lock:7771]: Oh, something must be wrong! 🫠 
I'll check it now.

Comment (masoudmanson) [reply yarn.lock]: Well, It is fixed 🤓

Comment (tihuan) [reply yarn.lock]: Awesomeeee thanks so much, Masoud 🎉 👏 !

**Pull request merged**